### PR TITLE
disable disruptek, add frosty, testes (disabled)

### PR DESCRIPTION
**Pull request by disruptek**
Discretion being the better part of valour and all that...

### CODE REVIEW

File: testament/important_packages.nim [hunk 1]
```diff
@@ -31,27 +32,34 @@ pkg1 "cligen", "nim c --path:. -r cligen.nim"
...
+#pkg1 "frosty", "nimble test", "https://github.com/disruptek/frosty"
 pkg1 "gara"
 pkg1 "glob"
 pkg1 "ggplotnim", "nim c -d:noCairo -r tests/tests.nim"
-# pkg1 "gittyup", "nimble test", "https://github.com/disruptek/gittyup"
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "gittyup", "nimble test", "https://github.com/disruptek/gittyup"
 pkg1 "gnuplot", "nim c gnuplot.nim"
-# pkg1 "gram", "nim c -r --gc:arc --define:danger tests/test.nim", "https://github.com/disruptek/gram"
-  # pending https://github.com/nim-lang/Nim/issues/16509
```
Comment (timotheecour) [inline testament/important_packages.nim:48]: * keep this `pending https://github.com/nim-lang/Nim/issues/16509` ?
* space after `#` (in all your edits)

File: testament/important_packages.nim [hunk 2]
```diff
@@ -19,7 +19,8 @@ pkg1 "BipBuffer"
 # pkg1 "blscurve" # pending https://github.com/status-im/nim-blscurve/issues/39
 pkg1 "bncurve"
 pkg1 "brainfuck", "nim c -d:release -r tests/compile.nim"
-pkg1 "bump", "nim c --gc:arc -r tests/tbump.nim", "https://github.com/disruptek/bump"
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "bump", "nimble test", "https://github.com/disruptek/bump"
```
Comment (timotheecour) [inline testament/important_packages.nim:23]: `"nimble test"` is the default, this should be:
```nim
# pending https://github.com/nim-lang/Nim/issues/16447
# pkg1 "bump"
```

ditto elsewhere

Comment (disruptek) [reply testament/important_packages.nim:23]: You can change it if you want.  #16509 is either merged or not, but irrelevant in any event.

**Comment by disruptek** [comment]
A month is long enough.

**Pull request closed without merging**